### PR TITLE
Updated  Syscall test script to upload results into database.

### PR DESCRIPTION
**Pull request by rmsivakanth**
Created database table for syscall test results and updated script to upload results into database.

### CODE REVIEW

**Comment by rmsivakanth** [comment]
```
Execution is verified.

11/01/2018 07:26:08 : [INFO ] ~~~~~~~~~~~~~~~TEST END : PERF-SYSCALL-BENCHMARK~~~~~~~~~~
11/01/2018 07:26:08 : [INFO ] CURRENT - PASS    - 1
11/01/2018 07:26:08 : [INFO ] CURRENT - FAIL    - 0
11/01/2018 07:26:08 : [INFO ] CURRENT - ABORTED - 0
11/01/2018 07:26:08 : [INFO ] Checking background cleanup jobs..
11/01/2018 07:26:08 : [INFO ] Removed 0 context files.
11/01/2018 07:26:08 : [INFO ] Cycle Finished.. TC-63662
11/01/2018 07:26:09 : [INFO ]
Test Results Summary
ICA test run on 11/01/2018 07:14:39
ARM Image under test OpenLogic : CentOS : 7.4 : latest
Total Executed TestCases        1 (       1 Pass,        0 Fail,        0 Abort)
Total Execution Time(dd:hh:mm) 0:0:11
XML file: TestConfiguration

#Sr. Test Name : Test Result : Test Duration (in minutes)
----------------------------------------------------
1. PERF-SYSCALL-BENCHMARK : PASS : 11.43


Logs can be found at .\TestResults\2018-01-11-07-14-31-5612


11/01/2018 07:26:09 : [INFO ] Creating 'Azure-63662-buildlogs.zip' from '.\TestResults\2018-01-11-07-14-31-5612'
11/01/2018 07:26:10 : [INFO ] R:\GitHub\LISAv2\Azure-63662-buildlogs.zip created successfully.
11/01/2018 07:26:10 : [INFO ] Copied : .\report\report_TC-63662.xml --> .\report\report_TC-63662-junit.xml
11/01/2018 07:26:10 : [INFO ] Analysing results..
11/01/2018 07:26:10 : [INFO ] PASS  : 1
11/01/2018 07:26:10 : [INFO ] FAIL  : 0
11/01/2018 07:26:10 : [INFO ] ABORT : 0
11/01/2018 07:26:10 : [INFO ] LISAv2 exits with code: 0
```

File: Testscripts/Windows/PERF-SYSCALL-BENCHMARK.ps1 [hunk 1]
```diff
@@ -127,8 +127,53 @@ collect_VM_properties
...
+                    for ($i = 1; $i -lt $finalResult.Count; $i++) {
+                        if ($finalResult[$i].test){
+                            $SQLQuery += "('$TestCaseName','$(Get-Date -Format yyyy-MM-dd)','$HostType','$HostBy','$HostOS','$GuestOSType','$GuestDistro','$GuestSize','$KernelVersion','$LisVersion','$cpuInfo','$($finalResult[$i].test)',$($finalResult[$i].avgReal),$($finalResult[$i].avgUser),$($finalResult[$i].avgSystem)),"
+                        } else {
+                            continue
+                        }
+                    }
+                    $SQLQuery = $SQLQuery.TrimEnd(',')
+                    LogMsg $SQLQuery
+                    $connection = New-Object System.Data.SqlClient.SqlConnection
```
Comment (lizzha) [inline Testscripts/Windows/PERF-SYSCALL-BENCHMARK.ps1]: use function UploadTestResultToDatabase in Framework.psm1

File: Testscripts/Windows/PERF-SYSCALL-BENCHMARK.ps1 [hunk 2]
```diff
@@ -127,8 +127,45 @@ collect_VM_properties
...
+                $password = $xmlConfig.config.$TestPlatform.database.password
+                $database = $xmlConfig.config.$TestPlatform.database.dbname
+                $dataTableName = $xmlConfig.config.$TestPlatform.database.dbtable
+                $TestCaseName = $xmlConfig.config.$TestPlatform.database.testTag
+                if ($dataSource -And $user -And $password -And $database -And $dataTableName) {
+                    $GuestDistro    = Get-Content "$LogDir\VM_properties.csv" | Select-String "OS type"| ForEach-Object{$_ -replace ",OS type,",""}
+                    if ($UseAzureResourceManager) {
+                        $HostType   = "Azure-ARM"
+                    } else {
+                        $HostType   = "Azure"
```
Comment (lizzha) [inline Testscripts/Windows/PERF-SYSCALL-BENCHMARK.ps1]: missed this one, as LISAv2 support ARM only, there's no need to set different value for HostType, set it to "Azure" only

**Comment by lizzha** [comment]
Not to merge until the last comment is addressed

File: Testscripts/Windows/PERF-SYSCALL-BENCHMARK.ps1 (hunk 2 above)
Comment (juhlee-microsoft) [reply Testscripts/Windows/PERF-SYSCALL-BENCHMARK.ps1]: This comment is still valid, @rmsivakanth

Comment (rmsivakanth) [reply Testscripts/Windows/PERF-SYSCALL-BENCHMARK.ps1]: fixed.

**Pull request merged**